scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/wdfn-8.yml

```diff
@@ -0,0 +1,48 @@
+WDFN-8-1EP_8x6mm_P1.27mm_EP6x4.8mm:
+  device_type: 'WDFN'
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/8L_WDFN_6x8mm_MN_C04172A.pdf'
+  ipc_class: 'qfn'
+  body_size_x:
+    nominal: 8.0
+    tolerance: 0
+  body_size_y:
+    nominal: 6.0
+    tolerance: 0
+  lead_width_min: 0.35
+  lead_width_max: 0.45
+  lead_len_min: 0.45
+  lead_len_max: 0.55
+  EP_size_x: 6
+  EP_size_y: 4.8
+  EP_num_paste_pads: [2, 2]
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 4
+  
+WDFN-8-1EP_8x6mm_P1.27mm_EP6x4.8mm_ThermalVias:
```

Comment:
You do not need two sections. The generators makes the 'regular' and 'thermalVias' variant from just one section that contains the `thermal_vias` key.

Reply:
Oh, I didn't know this. Thank you. Section without thermal vias is removed.